src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -164,6 +165,10 @@ export default {
     @apply tw-border-none;
     @apply tw-text-gray-3;
     @apply tw-bg-transparent;
+    @apply tw-inline-block;
+    @apply tw-w-72;
```

Comment:
I'm not really sure about this... we need a fixed width to show the ellipsis but I'm puzzled - I'll speak with Itzy to ask what she thinks about it. In the main time let me know your thoughts plz

Reply:
Could we set a max-width? thoughts?

Reply:
spoken with Itzi - we'll set the max width to 120px. Thank you for the recommendation